config/samples/go_example_loadtest.yaml

```diff
@@ -50,6 +50,12 @@ spec:
   # results:
   #   bigQueryTable: "example-project.foo.demo_dataset"
 
+  # timeout and ttl field only take numbers and converted to duration, the unit
+  # of duration is fixed as ns, for example: 1s = 1000000000 ns, the number
+  # below indicate timeout=30s, ttl=2min
+  timeout: 30000000000
+  ttl: 120000000000
```

Comment:
I was going to talk about this in our meeting :). I made TTL and Timeout as time.duration, unfortunately go 1.14 fixes the unit as ns, see (https://github.com/golang/go/issues/4712), and it doesn't take scientific notion when I played with it. 

Method I : Direct fill as duration 
Pro: fail fast 
Con: inconvenient to use, prone to human error, but it might be okay since we give templates

Method II : Direct fill as string, and parse to durations
Pro: you can have you own units, convenient to use
Con: format check depend on controller

Method III : Direct fill as int64, and convert to durations
Pro: more convenient than Method I to some extend
        fail fast

When I trying to explain the good and bad of three ways, I tend to agree that take the field as int64 could be the best for the scenario. I will change it. :)